packages/data/addon/models/metadata/table.js

```diff
@@ -94,4 +94,21 @@ export default class Table extends EmberObject {
    * @property {String} source - the datasource this metadata is from.
    */
   source;
+
+  /**
+   * @property {String[]} timeGrainIds - supported timegrains for a column
+   */
+  timeGrainIds = [];
+
+  /**
+   * @property {Object[]} timeGrains - timeGrain objects with id and display name
+   */
+  get timeGrains() {
+    return this.timeGrainIds.map(grain => {
+      return {
+        id: grain,
+        name: `${grain.substr(0, 1).toUpperCase()}${grain.substr(1)}`
```

Comment:
any reason not to use `upperFirst`?

Reply:
Just didn't want to pull in another dependency for a simple operation, but we're already using it elsewhere in navi-data so I'll just use it here too.